kolibri/logger/test/factory_logger.py

```diff
@@ -12,8 +11,6 @@ class Meta:
         model = models.ContentSessionLog
 
     user = factory.SubFactory(FacilityUserFactory)
-    content_id = uuid.uuid4().hex
-    channel_id = uuid.uuid4().hex
     start_timestamp = datetime.datetime.now()
```

Comment:
This shouldn't happen in production, although I think that the root topic of every channel has its content id and pk set to the channel id. But we should never log on that.